src/components/Note.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+
+class Note extends Component {
```

Comment:
If you don't use any local state, you can make that a functional component instead of class